### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by fsonnino**

### CODE REVIEW

File: wrappers/Fluent/README.rst [hunk 1]
```diff
@@ -5,72 +5,73 @@ Contributors
...
+Let us call the main directory where the Fluent case and the fluent wrapper is (coolprop/wrappers/Fluent) as CUSTOM_DIRECTORY.
 
 1. Make sure you are in the CUSTOM_DIRECTORY
 
 2. Make sure your case/mesh and UDF files are in the same directory as compile.sh (CUSTOM_DIRECTORY).
 
    a. If they are not, move your case/mesh and UDF files to CUSTOM_DIRECTORY. Do NOT move compile.sh.
    
-   b. By default, an UDF example (UDF.c and properties.c) has been included in CUSTOM_DIRECTORY.
+   b. By default, several UDF examples (CoolProp_Properties_of_Water.c OR CoolProp_Properties_of_Brine.c) has been included in CUSTOM_DIRECTORY, one and only must be choose
```
Comment (ibell) [inline wrappers/Fluent/README.rst]: "has been added" -> "have been added"
"choose" -> "chosen"

File: wrappers/Fluent/README.rst [hunk 2]
```diff
@@ -5,72 +5,73 @@ Contributors
...
-   a. Using the Fluent interface, go to Define > User-Defined > Functions > Compiled
-   
-   b. Select the .c files in CUSTOM_DIRECTORY as UDFs and .h files, if any, as headers
-   
-   c. Make sure "libudf" is written in the Library field
-   
-   d. Hit Load
-   
-7. The default UDF.c provided by the Coolprop wrapper is an EXECUTE_ON_DEMAND file.
+7. By default the udf examples ( for instance CoolProp_Properties_of_water.c ) embeded EXECUTE_ON_DEMAND
```
Comment (ibell) [inline wrappers/Fluent/README.rst]: should "embeded" be "embed"?

File: wrappers/Fluent/README.rst [hunk 3]
```diff
@@ -5,72 +5,73 @@ Contributors
...
 
-   a. To check if it is working: Define > User-defined > Execute on Demand
-   
-   b. Select "call_coolprop::libudf" and hit execute
+   a. Using the Fluent interface to check if it is working: Define > User-defined > Execute on Demand
+      
+   b. Select "call_coolprop_water::libudf" and hit execute
    
-8. The default properties.c provided integrates thermal conductivity, density, viscosity and specific heat from Coolprop library with the Fluent solver. Do not load UDF.c and properties.c simultaneously.
+8. The default CoolProp_Properties_of_Water.c provided integrates thermal conductivity, density, viscosity and specific heat from Coolprop library with the Fluent solver.
```
Comment (ibell) [inline wrappers/Fluent/README.rst]: Please always write CoolProp as CoolProp (watch the case)

File: wrappers/Fluent/README.rst [hunk 4]
```diff
@@ -5,72 +5,73 @@ Contributors
...
    
-   b. The fluid is, by default, carbon dioxide. To change the working fluid, you have to change the FLUID variable in the properties.c UDF file BEFORE running compile.sh. A full list of fluids supported by Coolprop may be found here: http://coolprop.sourceforge.net/Fluids/FluidInformation.html
+   b. The fluid is, by default, water. To change the working fluid, you have to change the FLUID[] variable in the CoolProp_Properties_of_Water.c UDF file BEFORE running compile.sh. A full list of fluids supported by Coolprop may be found here: http://www.coolprop.org/FluidInformation.html
+   Another example is given in CoolProp_Properties_of_Brine.c, the FLUID[] is a mixture between chloride sodium with a 20% concentration
    
-   c. If the operating pressure in your Fluent case is different than atmospheric (101325 Pa), you will also have to change gauge variable in properties.c
+   c. If the operating pressure in your Fluent case is different than atmospheric (101325 Pa), you will also have to change gauge variable in CoolProp_Properties_of_Water.c
```
Comment (ibell) [inline wrappers/Fluent/README.rst]: "gauge" -> "the gauge pressure"

File: wrappers/Fluent/README.rst (hunk 2 above)
Comment (fsonnino) [reply wrappers/Fluent/README.rst]: yes you're right

Comment (fsonnino) [reply wrappers/Fluent/README.rst]: it is embed

**Pull request merged**